src/Microsoft.Data/DataFrame.cs

```diff
@@ -45,18 +45,9 @@ public DataFrame(IList<DataFrameColumn> columns)
 
         public int ColumnCount => _table.ColumnCount;
```

Comment:
Do we still need `ColumnCount` when we have `.Columns.Count`?

Reply:
Since `RowCount` exists, I felt that a user might also expect to find `ColumnCount`.

Reply:
Well, to play devils advocate: We have a `Columns` property, but we don't have a `Rows` property.

Reply:
Alright, reasonably fair observation. Deleted `ColumnCount`